tests/integration/test_fim/test_files/test_audit/test_audit.py

```diff
@@ -301,7 +301,15 @@ def get_audit_creation_time():
                 return proc.create_time()
         pytest.fail("Auditd is not running")
 
-    plugin_path = "/etc/audisp/plugins.d/af_wazuh.conf"
+    audisp_path = "/etc/audisp/plugins.d/af_wazuh.conf"
+    audit_path = "/etc/audit/plugins.d/af_wazuh.conf"
+
+    if os.path.exists(audisp_path):
```

Comment:
Instead of checking for the existing path, we could run `auditctl -v` which shows the version of auditd that is available in the system:
```
vagrant@server:~$ sudo auditctl -v
auditctl version 2.8.5
```
We can split the line, grab the version and check whether it starts with 2 or 3, using the path that corresponds with the version and checking directly against it. This would also ensure that creating the path for version 3 on a system with audit v2 is accepted incorrectly.

Reply:
Fixed in 9758990